worlds/dlcquest/Regions.py

```diff
@@ -0,0 +1,331 @@
+import math
+import re
+from BaseClasses import MultiWorld, Region, Location, Entrance, ItemClassification
+from .Locations import DLCQuestLocation, location_table
+from .Rules import create_event
+from . import Options
+
+DLCQuestRegion = ["Movement Pack", "Behind Tree", "Psychological Warfare", "Double Jump Left",
+                  "Double Jump Behind the Tree", "The Forest", "Final Room"]
+
+
+def add_coin_freemium(region: Region, Coin: int, player: int):
+    name_coin = "number coins freemium"
+    number_coin = re.sub("number", str(Coin), name_coin)
+    location_coin = re.sub("number", region.name, name_coin)
```

Comment:
```suggestion
    number_coin = f"{Coin} coins freemium"
    location_coin = f"{region.name} coins freemium"
```

RegEx is generally slower than specialized string functions, doubly so if you don't precompile your patterns (re.compile)